src/chrome/content/rules/Adnxs.com.xml

```diff
@@ -1,58 +1,34 @@
 <!--
 	For other AppNexus coverage, see AppNexus.xml.
 
+	^adnxs.com doesn't exist
 
-	Problematic hosts in *adnxs.com:
-
-		- cdn ᴬ
-
-	ᴬ Akamai / mismatched
-
-
-	^adnxs.com doesn't exist.
-
-
-	Insecure cookies are set for these domains and hosts: ᶜ
+	Invalid certificate:
+		- www.adnxs.com
 
+	Insecure cookies are set for these domains and hosts (see https://owasp.org/index.php/SecureFlag):
 		- .adnxs.com
 		- www.adnxs.com
-
-	ᶜ See https://owasp.org/index.php/SecureFlag
-
 -->
-<ruleset name="Adnxs.com">
 
+<ruleset name="Adnxs.com">
 	<target host="*.adnxs.com" />
```

Comment:
The wildcard seems to break some sites. Can we be more restrictive?

Reply:
Changed it to explicitly list targets instead of using a wildcard